src/applications/financial-status-report/config/form.js

```diff
@@ -445,15 +445,36 @@ const formConfig = {
           title: 'Real estate',
           uiSchema: pages.realEstate.uiSchema,
           schema: pages.realEstate.schema,
+          depends: formData => !formData['view:enhancedFinancialStatusReport'],
         },
         realEstateRecords: {
           path: 'real-estate-asset-records',
           title: 'Real estate',
           uiSchema: pages.realEstateRecords.uiSchema,
           schema: pages.realEstateRecords.schema,
-          depends: ({ questions }) => questions.hasRealEstate,
+          depends: formData =>
+            formData.questions.hasRealEstate &&
+            !formData['view:enhancedFinancialStatusReport'],
           editModeOnReviewPage: true,
         },
+        enhancedRealEstate: {
+          path: 'enhanced-real-estate-assets',
+          title: 'Real estate',
+          uiSchema: pages.enhancedRealEstate.uiSchema,
+          schema: pages.enhancedRealEstate.schema,
+          depends: formData => formData['view:enhancedFinancialStatusReport'],
+          editModeOnReviewPage: false,
+        },
+        enhancedRealEstateRecords: {
+          path: 'enhanced-real-estate-asset-records',
+          title: 'Real estate',
```

Comment:
I have no context on this one but I was wondering if the title should be Real Estate Records?

Reply:
It could be and it likely will once we remove the old stuff after this goes through testing. The "enhanced" wording is kinda a placeholder across all of our enhanced components and stuffs.